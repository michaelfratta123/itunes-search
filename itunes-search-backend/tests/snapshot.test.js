const app = require("../app");
// create a snapshot test
test("Server snapshot", () => {
  expect(app).toMatchSnapshot();
});
