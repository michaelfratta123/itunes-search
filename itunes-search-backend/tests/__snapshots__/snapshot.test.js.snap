// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Server snapshot 1`] = `[Function]`;
